2020-01-30 16:45:29,416 ERROR: Exception on /register [GET] [in /home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/whisllayjr/Documentos/mymicroblog/app/routes.py", line 72, in register
    return render_template('register.html', title='Register', form=form)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/loaders.py", line 129, in load
    code = environment.compile(source, name, filename)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/whisllayjr/Documentos/mymicroblog/app/templates/register.html", line 2, in template
    {% import 'bootstrap/wtf.html' %}
jinja2.exceptions.TemplateSyntaxError: expected token 'as', got 'end of statement block'
2020-01-30 16:53:17,129 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:46]
2020-01-30 16:53:30,107 ERROR: Exception on /user/whisllaybastos [GET] [in /home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/whisllayjr/Documentos/mymicroblog/app/routes.py", line 86, in user
    next_url=next_url, prev_url=prev_url)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/loaders.py", line 129, in load
    code = environment.compile(source, name, filename)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/whisllayjr/Documentos/mymicroblog/app/templates/user.html", line 37, in template
    <li class="next{% not next_url %} disabled{% endif %}">
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'not'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2020-01-30 16:53:51,938 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:46]
2020-01-30 16:54:07,982 ERROR: Exception on /register [GET] [in /home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/whisllayjr/Documentos/mymicroblog/app/routes.py", line 72, in register
    return render_template('register.html', title='Register', form=form)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/loaders.py", line 129, in load
    code = environment.compile(source, name, filename)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/whisllayjr/Documentos/mymicroblog/venv/lib64/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/whisllayjr/Documentos/mymicroblog/app/templates/register.html", line 2, in template
    {% import 'bootstrap/wtf.html' %}
jinja2.exceptions.TemplateSyntaxError: expected token 'as', got 'end of statement block'
2020-01-30 16:54:47,288 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:46]
2020-01-30 17:18:33,320 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:48]
2020-01-30 17:26:47,158 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:48]
2020-01-30 18:38:43,715 INFO: Microblog startup [in /home/whisllayjr/Documentos/mymicroblog/app/__init__.py:50]
